app/filtering.js

```diff
@@ -303,7 +303,7 @@ function registerForHeadersReceived (session, partition) {
         continue
       }
       if (results.responseHeaders) {
-        cb({responseHeaders: results.responseHeaders})
+        cb({ responseHeaders: results.responseHeaders, statusLine: results.statusLine })
```

Comment:
This doesn't seem to actually redirect the browser to the given Location header. Instead, it loads `about-error.html` page. Any ideas?

<img width="1139" alt="screen shot 2017-02-21 at 6 19 07 pm" src="https://cloud.githubusercontent.com/assets/121766/23194347/61cadd08-f862-11e6-80b1-0455e884657a.png">